netty/src/main/java/io/grpc/transport/netty/NettyClientStream.java

```diff
@@ -67,6 +67,7 @@ public void request(final int numMessages) {
       @Override
       public void run() {
         requestMessagesFromDeframer(numMessages);
+        channel.flush();
```

Comment:
If the intent is to flush a windows update, I think this should be done by `NettyClientHandler` and `NettyServerHandler` returnProcessedBytes()`. WDYT?